### ISSUE TITLE
Guideline 2.12.7 User control - need for a requirement: not disable / disrupt  an accessibility feature

### ISSUE THREAD

**Issue by spanchang** [w3c/wcag3#410]
There is a requirement in the WCAG3 guidelines testing sheet  that reads: Content responds to users’ platform and user agent accessibility-related settings.
But I believe this does not say that "content should not disable / disrupt  an accessibility feature provided by the platform". This should be an explicit requirement IMO. 
For instance, ability to enter text using handwriting mode (as a VoiceOver user, I do that sometimes on an iPhone for search or login fields), or ability to use the dictate feature is sometimes disabled /not available. I have noticed this in native apps and less frequently on webpages.
Thanks
